tools/scripts/buildAndTest.cmd

```diff
@@ -0,0 +1,17 @@
+rem Start at the root of the Catch project directory, for example:
+rem cd Catch2
+
+rem begin-snippet: catch2-build-and-test
```

Comment:
```suggestion
rem begin-snippet: catch2-build-and-test-win
```

The snippet's "name" needs to be different from the other one, because they should be unique (so that embedding snippets into docs is not ambiguous).